src/python_pachyderm/client.py

```diff
@@ -198,14 +231,18 @@ def new_from_pachd_address(
         )
 
     @classmethod
-    def new_from_config(cls, config_file):
-        """
-        Creates a Pachyderm client from a config file-like object.
-
-        Params:
-
-        * `config_file`: A file-like object containing the config
-        json file.
+    def new_from_config(cls, config_file: StringIO) -> "Client":
```

Comment:
I think instead of `io.StringIO`, we should use one of the concrete types from the `typing` module. In this case `typing.IO` or `typing.TextIO` makes sense.

Reply:
updated